functional-samples/cookbook.file_handlers/README.md

```diff
@@ -0,0 +1,25 @@
+# Cookbook - File Handling API
+
+This sample demonstrates how to use the File Handling API in an extension.
+
+## Overview
+
+On ChromeOS only, extensions can use the `file_handlers` manifest key to
+register as a file handler for particular file types. This behaves in the same
+way as the
+[equivalent API](https://developer.chrome.com/articles/file-handling/) in web
+applications.
+
+With this extension installed, you can open .txt files with the extension and see their name and size on the opened extension page. This could be a good starting point for building an extension which displays or interacts with an opened file.
```

Comment:
```suggestion
With this extension installed, you can open text files with the extension and see their name and size on the opened extension page. This could be a good starting point for building an extension that displays or interacts with an opened file.
```